### ISSUE TITLE
Get admin authkey for automatic testing of MISP install

### ISSUE THREAD

**Issue by Rafiot** [MISP/MISP#644]
It would be a nice to have in order to be able to use the API to automatically test a fresh install (also travis :) )

**Comment by iglocska** [comment]
Imagine all the admins that would not change it though...

**Comment by Rafiot** [comment]
True... and changing the password doesn't change the auth key, right? (is it a good thing?)

**Comment by Rafiot** [comment]
Let's do it differently: how can I create an account without trying to connect?

Then, something like that will do the job: 
```
select authkey from users where email = "admin@admin.test";
```

**Comment by Rafiot** [comment]
Fixed by 3feeee920e2fbaf78abb3f7248d36694fdeea29b
